Cargo.toml

```
@@ -53,7 +53,7 @@ thiserror = "1.0.21"
 toml = "0.7.2"
 tower = "0.4"
 url = "2.2.0"
-wasm-bindgen = { version = "0.2.83", features = ["serde-serialize"] }
+wasm-bindgen = { version = "=0.2.86", features = ["serde-serialize"] }
```

This PR shouldn't modify this dependency afaik.

```suggestion
wasm-bindgen = { version = "0.2.83", features = ["serde-serialize"] }
```

---

I'd argue "why does the secp256r1 builtin function require a bump to `wasm-bindgen`?"

Generally we want to avoid changing versions of packages in PRs which are unrelated to the dependency in question.

---

Okay I see what you mean, I agree that it is orthogonal to this PR.

This PR won't pass CI (build-was) without it, so let me see if it can be reproduced in a separate branch

---

Note that the version in the lockfile was `0.28.7`.  I'll open a PR to first lock this version down, since the ABI is brittle between even minor version (as seen in the CI failing)

---

Opened a PR here: https://github.com/noir-lang/noir/pull/1898 to pin the lockfile to 0.28.6